discovery-infra/test_infra/consts/consts.py

```diff
@@ -158,3 +170,12 @@ class Platforms:
 class HighAvailabilityMode:
     FULL = 'Full'
     NONE = 'None'
+
+
+class BaseAsset:
+    MACHINE_CIDR = "192.168.127.0/24"
```

Comment:
why 127? i think our default is 126 no?

Reply:
to avoid conflicts in case someone runs discovery flow and then tests (or in parallel)

Reply:
no one will run discovery flow and tests in parallel, it is not supported.
In any other case we should not fail. Lets have same starting point as before

Reply:
https://github.com/openshift/assisted-test-infra/pull/806#discussion_r648935526

Reply:
see run operator - we deploy operator and then run tests against it